ConditionsConsumed/runCMSDrivers_mc2018.sh

```diff
@@ -2,7 +2,7 @@
 echo "------------------------------"
 echo "Running step-1/9: GEN"
 echo "------------------------------"
-cmsDriver.py TTbar_13TeV_TuneCUETP8M1_cfi  --conditions 105X_upgrade2018_realistic_v6 -n 5 --era Run2_2018 -s GEN --fileout output_step1_GEN.root --beamspot Realistic50ns13TeVCollision  --customise_commands='process.GlobalTag.DumpStat =cms.untracked.bool(True)'|& tee output_step1_GEN.log
+cmsDriver.py TTbar_13TeV_TuneCUETP8M1_cfi  --conditions auto:phase1_2022_realistic -n 5 --era Run2_2018 -s GEN --fileout output_step1_GEN.root --beamspot Realistic50ns13TeVCollision  --customise_commands='process.GlobalTag.DumpStat =cms.untracked.bool(True)'|& tee output_step1_GEN.log
```

Comment:
I think this should be kept as `--conditions 105X_upgrade2018_realistic_v6`

Reply:
You are right, i change this file (just to make some tests), and forgot to change again. I will fix that